haiku-apps/voxophone/voxophone-1.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="An online radio player"
+DESCRIPTION="Voxophone is a online radio player application for Haiku."
+HOMEPAGE="https://github.com/aarroyoc/Voxophone"
+COPYRIGHT="2015 Adrian Arroyo Calle"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="67514047957bb8a186673ccc6fa71d2d0a1e1728"
+SOURCE_URI="https://github.com/aarroyoc/Voxophone/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="bc98a70248057aa482f038e3689876dc0632aca4aa65d2417d9f692c86b486b5"
+SOURCE_FILENAME="voxophone-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Voxophone-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	voxophone = $portVersion
+	app:Voxophone = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libnetapi
```

Comment:
`lib:libnetapi` is a part of `haiku`, so you don't have to specify it

```diff
-	lib:libnetapi

Reply:
@alaviss, you are right!

For some unknown reason, when I tried to build the recipe (before makefile_engine was removed), I got an error about lnetapi, so I typed:
```
pkgman search lib:libnetapi
Status  Name       Description                                 
---------------------------------------------------------------
s       samba      Library bits of the samba network filesystem
s       samba_x86  Library bits of the samba network filesystem
```
and thought it needed samba.
But now, without `makefile_engine` and without `lib:libnetapi` the recipe builds fine on x86_gcc2 :)

So sorry @bach5000 for my incorrect suggestion and thanks for your help!

(I'll try on x86_64 soon.)

Reply:
Hmm, on x86_64 it fails to build:
```
Scanning dependencies of target Voxophone
[ 25%] Building CXX object CMakeFiles/Voxophone.dir/src/app.cpp.o
[ 50%] Building CXX object CMakeFiles/Voxophone.dir/src/dashboard.cpp.o
[ 75%] Building CXX object CMakeFiles/Voxophone.dir/src/player_manager.cpp.o
[100%] Linking CXX executable Voxophone
/boot/system/develop/tools/bin/../lib/gcc/x86_64-unknown-haiku/5.4.0/../../../../x86_64-unknown-haiku/bin/ld: cannot find -lnetapi
collect2: error: ld returned 1 exit status
CMakeFiles/Voxophone.dir/build.make:146: recipe for target 'Voxophone' failed
```
What's funny is that on x86_64 it says it `cannot find -lnetapi` but on x86_gcc2 it did not link against it:
```
objdump -x /system/apps/Voxophone |grep -w NEEDED
  NEEDED      libbe.so
  NEEDED      libroot.so
  NEEDED      libtracker.so
  NEEDED      libmedia.so
  NEEDED      libnetwork.so
  NEEDED      libbnetapi.so
  NEEDED      libstdc++.r4.so
```
Hopefully, making this change in CMakeLists.txt:
```diff
-TARGET_LINK_LIBRARIES(Voxophone be root tracker media network netapi)
+TARGET_LINK_LIBRARIES(Voxophone be root tracker media network bnetapi)
```
allows to build on x86_64 :)

BTW, I added a debuginfo package with:
```
defineDebugInfoPackage voxophone \
	$appsDir/Voxophone
```
and it is creating a real package, not a fake one :)